Tryouts/Visuals/Windows/WindowsVisualUtils/EmbeddedWebView.cs

```diff
@@ -0,0 +1,166 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Platform;
+using Microsoft.Web.WebView2.Core;
+using Microsoft.Web.WebView2.WinForms;
```

Comment:
Why Windows Forms? Why not WPF?

Reply:
WinForms controls have the Window Handle automatically - they do not require an extra step for obtaining the handle. It does not matter what we use for the WebView2 browser - the browser/shell communications are exactly the same.

Reply:
It could matter, and perhaps also be beneficial to use winforms.  With cefsharp we found with winforms we could have 60fps vs 30fps of wpf hosting and there were also less issues with keyboard focus, etc.